lib/block_view.go

```diff
@@ -109,6 +109,11 @@ type UtxoView struct {
 	// Map of PKID to the next nonce to use for a transaction.
 	PKIDToNextNonce map[PKID]uint64
 
+	// Map of TxHash to filled order. Needed to handle
+	// derived key accounting for DAOCoinLimitOrders
+	// where the transactor is selling DESO.
+	TxHashToFilledOrder map[BlockHash][]*FilledDAOCoinLimitOrder
```

Comment:
nit: naming TxHashToFilledOrder --> TxHashToFilledDAOCoinLimitOrder. Would it be possible that we would have other types of orders in the future? CC orders for example. More expicit ~ more future-proof. :man_shrugging: